script.video.nfl.gamepass/addon.xml

```diff
@@ -15,17 +15,17 @@
   <extension point="xbmc.addon.metadata">
     <summary lang="en_GB">NFL Game Pass</summary>
     <description lang="en_GB">Watch NFL Game Pass streams.</description>
+    <lifecyclestate>broken</lifecyclestate>
```

Comment:
Please carefully read the docs for `<lifecyclestate>` tag and fix it.

Reply:
Sorry, I shouldn't code when distracted...